### DISCUSSION TITLE
Issue with Creating a New Application on Etsy

### DISCUSSION THREAD

**Discussion by WinAx** [etsy/open-api#1069] [Q&A]
I am trying to create a new app on Etsy, but I am getting an error message: "An unknown error occurred. Please visit [this page](https://developers.etsy.com/documentation/get-help) for assistance." I have already visited the page, but I was unable to find any information that would help me resolve the error.

I would be very grateful if you could help me to fix this issue.

Thank you for your time.

**Comment by maximst** [comment]
I have the same problem. @WinAx are you resolved this error?
